internal/services/zero_trust_tunnel_warp_connector/data_source_test.go

```diff
@@ -1,36 +1,40 @@
 package zero_trust_tunnel_warp_connector_test
 
 import (
-	"errors"
+	"fmt"
+	"os"
 	"testing"
 
 	"github.com/cloudflare/terraform-provider-cloudflare/internal/acctest"
 	"github.com/cloudflare/terraform-provider-cloudflare/internal/utils"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
-	"github.com/hashicorp/terraform-plugin-testing/terraform"
 )
 
-func TestAccCloudflareZeroTrustTunnelWarpConnectorDataSource_Basic(t *testing.T) {
+func TestAccWARPConnectorDatasourceBasic(t *testing.T) {
+	if os.Getenv("CLOUDFLARE_API_TOKEN") != "" {
```

Comment:
Is this unset for the same reason? https://github.com/cloudflare/terraform-provider-cloudflare/pull/5695/files

Reply:
Yes, but I don't think the reason is true. The issue is that in our openapi spec we only have the authentication mechanism of the API Key and not the API Token. So, the code assumed we don't support it. When we actually support. Since it doesn't hurt us, I just kept it as it is.